### ISSUE TITLE
Add modal for Admin to notify user when "done" commenting

### ISSUE THREAD

**Issue by stephaniesimms** [DMPRoadmap/roadmap#728]
UX wireframe @rhu001 http://h90c3m.axshare.com/admin_finish_review_dialog_box.html

Once admin follows notification link to the write plan page for the plan that they are going to provide feedback on, whenever they attempt to leave the write plan tab, they will see the modal box pop up asking them if they are "done" commenting and ready to send notification back to user or if they just want to save existing comments and return later. 

Since comments are autosaved, revise the modal text as follows:
"No, I still have some more work to do. I'll finish providing feedback at a later time."

When an admin clicks "Send" the system should trigger the following email:

Email title: [TOOL NAME]: Expert feedback has been provided for [PLAN TITLE]

Hello [USER NAME]

[COMMENTER NAME] has finished providing feedback on the plan [“PLAN TITLE”]. To view the comments, please visit the My Dashboard page in [TOOL NAME] and open your plan. 

All the best
The [TOOL NAME] team

You may change your notification preferences on your profile page. Please do not reply to this email. If you have any questions or need help, please contact us at [HELPDESK EMAIL] or visit [CONTACT US URL]

**Comment by briri** [comment]
See #727 for Dev notes and the workflow for the Feedback process

**Comment by briri** [comment]
Need to clarify text on modal.

**Comment by stephaniesimms** [comment]
Since comments are autosaved, you can revise the modal text as follows:
"No, I still have some more work to do. I'll finish providing feedback at a later time."

**Comment by jollopre** [comment]
FYI @stephaniesimms, comments are not auto-saved, you have to click save/update otherwise they are not persisted.

**Comment by stephaniesimms** [comment]
ok @jollopre - if there will be no autosave functionality for comments then the modal text is fine as is. no changes necessary.

**Comment by briri** [comment]
A JS solution for this would not be ideal since we're doing auto saves and do not have a good way to trigger a modal popup. Wiring one to the comment's auto-save JS would be too noisy.

Spoke with @stephaniesimms and we decided to just add a 'Complete' link to the Org Admin -> Plans notification section (see screenshot). The Org Admin accesses the plan via this page, so they will see it before starting the feedback process. The 'Complete' link has a tooltip that reads: 'Notify the plan owner that I have finished providing feedback'
![screen shot 2017-11-22 at 9 38 59 am](https://user-images.githubusercontent.com/1204467/33151216-fcb77abe-cf8b-11e7-8f61-bb1477c3dab0.png)
